packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -416,7 +420,7 @@ - (void)updatePlayingState {
   } else {
     [_player pause];
   }
-  _displayLink.running = _isPlaying;
+  _displayLink.running = _isPlaying || self.waitingForFrame;
```

Comment:
can you also add a comment here about the paused case?

Reply:
Done.